quartz_solar_forecast/forecasts/v1_1.py

```diff
@@ -0,0 +1,43 @@
+import pandas as pd
```

Comment:
Im not sure I like the name `v1_1` perhaps `v1_tilit_orientation ` would be slightly more verbose

Reply:
makes sense, I'll update it.